mobile-ios/readme.MD

```diff
@@ -11,8 +11,8 @@ Once you've gone through this and sent us some answers, we'll have someone revie
 # Questions
 - What's your proudest achievement? Something you've done in work, some random side project, code or otherwise. Just make sure you explain what it is that makes you so proud of it.
 - What's the most interesting technical article you've read recently? What did you like about it and why should we take a look at it?
-- Have a look at this Xcode playground (open with latest Xcode 8): https://gist.github.com/roland9/adac6de865314244c0581cb8ad21010c/ec6fc5079ba4569c6b5c61b1026189bf4e23e18a
-  The code creates a Table View controller showing a list of Artists and Albums.
+- Write some code that will flatten an array of arbitrarily nested arrays of integers into a flat array of integers. e.g. [[1,2,[3]],4] -> [1,2,3,4]. This is an opportunity to show us your understanding of computer science fundamentals. Avoid using built-in language methods like Array.flatten.
```

Comment:
Complete nitpick, but can you wrap Array.flatten in backticks, like so - `Array.flatten`. Also maybe reference the relevant Swift method.